src/lib/index/index_cp.cpp

```diff
@@ -63,8 +63,8 @@ void IndexCPContext::add_to_txn_journal(uint32_t index_ordinal, const IndexBuffe
     }
 }
 
-void IndexCPContext::add_to_dirty_list(const IndexBufferPtr& buf) {
-    m_dirty_buf_list.push_back(buf);
+void IndexCPContext::add_to_dirty_list(const IndexBufferPtr& buf, const BtreeNodePtr& node) {
+    m_dirty_buf_list.push_back(std::make_pair(buf, node));
```

Comment:
The node ref_cnt is automatically increased in its constructor, right?

Reply:
Yes, the unit test actually fails without this.
Sure, I will add the comment.